synthesis/output.py

```diff
@@ -271,3 +274,19 @@ def execute(context):
     if "geoparquet" in output_formats:
         path = "%s/%strips.geoparquet" % (output_path, output_prefix)
         df_spatial.to_parquet(path)
+    
+    # Output population analysis
+    SAMPLING_RATE =context.config("sampling_rate")
+    df_spatial = df_spatial.to_crs("EPSG:2154")
+
+    df_spatial["distance"] = df_spatial.length/1000
+    df_spatial["distance_class"] = pd.cut(df_spatial["distance"],list(np.arange(100))+[np.inf])
+
+    # Compare distance 
+    analysis_distance = context.stage("analysis.synthesis.population")
+    analysis_distance = pd.concat([analysis_distance,df_spatial.groupby("distance_class")["person_id"].count()],axis=1).reset_index()
+    analysis_distance.columns = ["Distance class","HTS","EQASIM"]
+    analysis_distance["Proportion_HTS"] = analysis_distance["HTS"] / analysis_distance["HTS"].sum()
+    analysis_distance["Proportion_EQASIM"] = analysis_distance["EQASIM"] / len(df_spatial)
+    analysis_distance["EQASIM"] = analysis_distance["EQASIM"]/ SAMPLING_RATE
+    analysis_distance.to_csv(f"{output_path}/{ANALYSIS_FOLDER}/{output_prefix}distance.csv")
```

Comment:
Hi Marie, I would prefer if this part is also generated rather by activating a stage in the configuration, like `analysis.distances` or something, for instance. I'd like to stick the `synthesis.output` to generating the output data, and all the analysis apart.

Reply:
Hi Sebastian, I will change it right away to make analysis apart of output data and note it for future addition of analysis. But could it be added a parameter to the `synthesis.output ` to permit generation of analysis file automatically with other output data or should just the docs be update in section _Analysing synthetic population_ with the list of new analysis ?